.github/workflows/shared-increase-version-number.yaml

```diff
@@ -56,17 +60,17 @@ jobs:
       - name: Update ${{ inputs.working_dir }}/package-lock.json
         id: updatePackageLockJson
         run: |
-          npm install
+          npm install --package-lock-only
 
       - name: Commit ${{ inputs.working_dir }}/package.json, ${{ inputs.working_dir }}/package-lock.json and ${{ inputs.working_dir }}/version.json
-        if: ${{ inputs.version_number_input == '' && steps.checkRelease.outputs.IS_RELEASE != 'true' }} 
+        if: ${{ inputs.version_number_input == '' && steps.setup.outputs.IS_RELEASE != 'true' }} 
```

Comment:
what's the purpose of changing this?

Reply:
I messed the branch up with a force push: https://github.com/IABTechLab/uid2-shared-actions/commit/4096e54e0d97b3a128dd8a7a7692e790c20f2cf0 This is to revert the change

Reply:
gotcha, thanks!